modules/apps/app-builder/app-builder-test/src/testFunctional/macros/FormViewBuilder.macro

```diff
@@ -1077,6 +1077,12 @@ definition {
 			locator1 = "FormViewBuilder#CHECKBOX_AT_FORM_BODY");
 	}
 
+	macro viewEntryTitle {
+		AssertElementPresent(
+			key_text = "${key_titleName}",
+			locator1 = "FormViewBuilder#ENTRIES_TITLE_CONTENT");
```

Comment:
@fmarcio, please rename:

- `viewEntryTitle` to `viewEntriesTitle`
- `key_text` and `key_titleName` to `key_entriesTitle`
- `ENTRIES_TITLE_CONTENT` to `ENTRIES_TITLE`

The goal behind this renaming is keep a consistent naming. :wink:

Reply:
@natocesarrego done most of the requests. I wasn't able to do some of the name changes requests because the locator FormViewBuilder#ENTRIES_TITLE_CONTENT is already merged on master. But the test is working normally :-)